### PR TITLE
Updates to .travis-ci.yml

### PR DESCRIPTION
**Pull request by omus**
Moves the AWS credential information into the .travis-ci.yml along with running Travis CI on 1.1 and nightly.

### CODE REVIEW

**Comment by omus** [comment]
Note that at this time the Travis CI settings with the credentials have been removed.

**Comment by omus** [comment]
bors try

File: .travis.yml [hunk 1]
```diff
@@ -5,22 +5,24 @@ os:
...
-matrix:
-  allow_failures:
-    - julia: nightly
+  - secure: I7/rE1u2w6kNlRiKsHGRonjC2Jsi/qlbHKNaLb5dH+K+YtDZo91J3WN4DRypKueWZAAFjYhgpHVm5Drt3TeKhQe9EF836DK7Mr6KBitXHp8S1xloCdnDl+w94j4wUxc5U48pCE3K0MncA7xIucbuHYyDigCzltQ7QRt32/6QmNOfe/FLLrxIs/kSuaugrt+XUfARrNbizaUtZfenJd3gC9VxFstjqKR+J5s4VWsrkS8xahlmME73pxuJW5e3unsBbWFs2oqRAxk9WG43wE96dIxD65MtHS+vO6C3157QDy/l67up2UtMNzboMoflPor7ZewOnslHXp1gl/3pEn50WJOQ3/jFA7Eb/sdQVl5Fd+9EO/vSwhOX4redtXPTwnzBTTxC4do6QTNRKJtyfRo6uKr1t48IYZZdBweGC7iHY911qtpxdUyqNRz/Lldrw/bV319TQWtyJYM+NBSzQ2ZJc33KlzzseBTO+YAXKyNTMAk5qyxjXnzKhZRT7RkyzPlKmngH2lfsU6eMsCCTB2sTF1dffz9uqVZ9bQYVd/yw/+IeVu/XZlT4T6vWfbp29CIhjHXOpprXGwQPhm8DKpiXtuGKYnmlpaR7wB1Gr3a9O0qgwEbYhj3x9BwvP9mrp6/D87jwDKG5aPv6N4U06KPf0sbfrziJx4Ooz8TEPPI8BjA=
```
Comment (iamed2) [inline .travis.yml]: should be indented again

Comment (omus) [reply .travis.yml]: Thanks.  The `travis` command line tool doesn't care about what formatting you were using...

**Comment by omus** [comment]
bors try

**Pull request merged**